tests/drivers/pwm/pwm_gpio_loopback/Kconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "PWM GPIO loopback test"
+
+source "Kconfig.zephyr"
+
```

Comment:
add soc/board specific configs here, along with default values if needed rather than having those in source code :)

Reply:
DEFAULT_PERIOD_CYCLE for example